packages/examples/src/tex.meta

```diff
@@ -20,6 +20,14 @@
     "fps": 60,
     "resolutionScale": 2,
     "colorSpace": "srgb",
+    "exporter": {
```

Comment:
Aside: This probably warrants a GitHub action that runs `vite` on each of the projects, loads the project, and makes sure that the meta file is fully up to date so that these don't get sucked into unrelated changes. I'll add that to my to-do list.